### ISSUE TITLE
./fips open on linux

### ISSUE THREAD

**Issue by mgerhardy** [floooh/fips#29]
[ERROR] don't know how to open a 'Unix Makefiles' project

even though i configured cmake for eclipse - is there an option to change that to also get support for eclipse?

**Comment by mgerhardy** [comment]
I mean, the workaround (by directly using cmake) also works - but as fips has an open command...

**Comment by floooh** [comment]
Yes I wasn't sure how to this since there are so many IDE options on Linux (e.g. I prefer QtCreator).

First we need to figure out what IDEs to support, and whether and how they can started to load a project at start (e.g. on OSX on Windows this is simple, just run 'open bla.xcodeproj' or 'start bla.sln', and the OS will start the application associated with the file type).

Next, there should be a './fips set ide' settings command to choose the preferred IDE, or alternatively figure it out from the cmake generator if cmake has direct support for it (like with Eclipse).

**Comment by mgerhardy** [comment]
Then this would most likely be solved by providing mimetype configurations for the system and just call xdg-open. That would most likely go hand in hand with "provide fips as tool (or whatever the issue name was)" - because It can then be packaged into the distribution and provide proper configurations.
